update container shorthand (#20884)
mdn/content: files/en-us/web/css/css_container_queries/index.md

  - : Defines an element as a **query container**. Descendants can query aspects of its sizing, layout, style and state.
- `container-name`
  - : Specifies a list of **query container names** for `@container` rules to use to filter which query containers are targeted.

> - `container`: `<'container-name'> [ / <'container-type'> ]?`

Reviewer: ```suggestion
- `container`: 
```

>   - : A shorthand property to set both `container-name` and `container-type` (use slash as separate sign).

Reviewer: ```suggestion
  - : A shorthand property to set both `container-name` and `container-type`. The syntax is `<'container-name'> [ / <'container-type'> ]?`,  with a slash between values when both are present.
```

- `container-name`
  - : Specifies a list of **query container names** for `@container` rules to use to filter which query containers are targeted.
- `container`:

>   - : A shorthand property to set both `container-name` and `container-type`. The syntax is `<'container-name'> [ / <'container-type'> ]?`,  with a slash between values when both are present.

Reviewer 2: ```suggestion
  - : A shorthand property to set both `container-name` and `container-type`. The syntax is `<'container-name'> [ / <'container-type'> ]?`, with a slash between values when both are present.
```
